### PR TITLE
Add Instructions for rubyx

### PR DESCRIPTION
**Pull request by postcumer**

### CODE REVIEW

File: 11/rubyx.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+### Pre-installation:
+
+* Boot image (from download page)
+* Gapps package (optional) (from download page, gapps button)
+
+### First time installation (clean flash):
+
+* Reboot in recovery
+* Format data
+* Install rom using adb sideload - ```adb sideload crDroid-Rom.zip```
```
Comment (gwolf2u) [inline 11/rubyx.md]: set command on new line with a blank row

File: 11/rubyx.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+* When asked to reboot to recovery, choose 'Yes' if you want to install gapps or 'No' if you don't want gapps and want to reboot to system
+* Now if you choosed to install gapps, simply sideload crDroid-gapps.zip the same way you installed crDroid-Rom.zip then reboot to system
+* Install gapps (optional) - ```adb sideload crDroid-gapps.zip```
```
Comment (gwolf2u) [inline 11/rubyx.md]: same here

File: 11/rubyx.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+* Start system
+
+### Update installation:
+#### Via recovery (recommended way):
+* Boot to recovery
+* Do ```adb sideload crDroid-Rom.zip```
```
Comment (gwolf2u) [inline 11/rubyx.md]: and here

File: 11/rubyx.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+* Reinstall gapps (optional) - ```adb sideload crDroid-gapps.zip```
```
Comment (gwolf2u) [inline 11/rubyx.md]: and ofc here

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: 11/rubyx.md (hunk 4 above)
Comment (postcumer) [reply 11/rubyx.md]: fixed them

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**